packages/checkout-common/package.json

```diff
@@ -6,6 +6,9 @@
   "module": "dist/esm/index.js",
   "types": "./dist/index.d.ts",
   "scripts": {
+    "build": "rollup -c",
```

Comment:
Something is wrong because these scripts are already there on main: https://github.com/saleor/react-storefront/blob/382c0a3c484cf2c56a8952eeda0f53b56121d3ad/packages/checkout-common/package.json#L11

Reply:
my bad, I didn't resolve the conflicts properly